src/platform/w3dengine/client/w3dpropbuffer.cpp

```diff
@@ -0,0 +1,239 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief Prop Drawing Code
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "w3dpropbuffer.h"
+#include "assetmgr.h"
+#include "baseheightmap.h"
+#include "camera.h"
+#include "geometry.h"
+#include "globaldata.h"
+#include "light.h"
+#include "lightenv.h"
+#include "partitionmanager.h"
+#include "playerlist.h"
+#include "rinfo.h"
+#include "w3dshroud.h"
+
+W3DPropBuffer::W3DPropBuffer()
```

Comment:
`m_propsUpdated`, `m_redoCull` not initialized.

Reply:
fixed.